crates/hive-utils/Dockerfile

```diff
@@ -0,0 +1,65 @@
+# TODO: Use official madara image when https://github.com/keep-starknet-strange/madara/issues/982 is closed
+# FROM ghcr.io/keep-starknet-strange/madara:v0.1.0.experimental.3 as madara
+FROM fredtupas/madara as madara
+
+FROM ghcr.io/kkrt-labs/kakarot/deployer:latest as deployer
+
+FROM ghcr.io/kkrt-labs/kakarot-rpc/node:latest
+
+# Install svn to download Madara genesis config files
+RUN apt-get -y update; \
+    apt-get install -y --no-install-recommends subversion; \
+    apt-get autoremove -y; \
+    apt-get clean; \
+    rm -rf /var/lib/apt/lists/*
+
+# Copy Madara binary
+COPY --from=madara /madara-bin /madara-bin
+
+# Madara Path
+ENV MADARA_PATH=/root/.madara
+ENV MADARA_CONFIG_PATH=${MADARA_PATH}/configs
+ENV KAKAROT_CONTRACTS_PATH=${MADARA_PATH}/cairo-contracts/kakarot
+
+# Download Madara genesis config to source path
+RUN svn export https://github.com/keep-starknet-strange/madara/trunk/configs ${MADARA_CONFIG_PATH}
+
+# Copy combined genesis.json to source path
+COPY .hive/genesis.json ${MADARA_CONFIG_PATH}/genesis-assets/genesis.json
+
+# Copy compiled Kakarot contracts to destination path of compiled contracts
+COPY --from=deployer /app/kakarot/build ${KAKAROT_CONTRACTS_PATH}
```

Comment:
Here we take the latest compiled contracts from Kakarot deployer, we download the configs from Madara and we move the combined `genesis.json` to the appropriate paths for Madara to use when initializing

Reply:
are we sure the genesis we generated by running make dump-genesis is using the latest kakarot contracts in the ci?